sources/packages/backend/apps/queue-consumers/src/processors/schedulers/sfas-integration/_tests_/sims-to-sfas-integration.scheduler.e2e-spec.ts

```diff
@@ -135,6 +137,24 @@ describe(describeProcessorRootTest(QueueNames.SIMSToSFASIntegration), () => {
         },
       );
 
+      const legacyRestriction = await db.restriction.save({
+        restrictionType: RestrictionType.Provincial,
+        isLegacy: true,
+        restrictionCode: "LGCY_XYZ",
+        notificationType: RestrictionNotificationType.Error,
+        restrictionCategory: "Other",
+        description: "Description for LGCY_XYZ",
+        actionType: [RestrictionActionType.StopPartTimeDisbursement],
+        updatedAt: simsDataUpdatedDate,
+      });
+
+      // Student has a legacy restriction that should be ignored.
+      await saveFakeStudentRestriction(db.dataSource, {
+        student,
+        application,
+        restriction: legacyRestriction,
+      });
+
```

Comment:
Please set the updated date of legacy restriction to fall under the delta period

```ts
// Student has a legacy restriction that should be ignored.
      const legacyStudentRestriction = await saveFakeStudentRestriction(
        db.dataSource,
        {
          student,
          application,
          restriction: legacyRestriction,
        },
      );

      await db.studentRestriction.update(
        {
          id: legacyStudentRestriction.id,
        },
        {
          updatedAt: simsDataUpdatedDate,
        },
      );
```